runtime/vm/bindnatv.cpp

```diff
@@ -957,17 +963,33 @@ bindNative(J9VMThread *currentThread, J9Method *nativeMethod, char * longJNI, ch
 		++argCount;
 	}
 
-	/* Search each shared library in the class loader for a matching native */
-	nativeLibrary = classLoader->librariesHead;
-	while (nativeLibrary != NULL) {
-		UDATA rc = lookupNativeAddress(currentThread, nativeMethod, nativeLibrary, longJNI, shortJNI, argCount, bindJNINative);
-		if (J9_NATIVE_METHOD_IS_BOUND(nativeMethod)) {
-			return J9_NATIVE_METHOD_BIND_SUCCESS;
-		} else if (J9_NATIVE_METHOD_BIND_OUT_OF_MEMORY == rc) {
-			return rc;
+#if JAVA_SPEC_VERSION >= 15
+	if (classLoader == vm->systemClassLoader) {
```

Comment:
Please move the braces out of the ifdef (similarly below).

Reply:
Fixed.